src/masternode/masternode-sync.cpp

```diff
@@ -16,12 +16,6 @@
 class CMasternodeSync;
 CMasternodeSync masternodeSync;
 
-void CMasternodeSync::Fail()
-{
-    nTimeLastFailure = GetTime();
-    nCurrentAsset = MASTERNODE_SYNC_FAILED;
```

Comment:
Now that whole `MASTERNODE_SYNC_FAILED` state got impossible to get into, so all the code that handles the state can be removed as well. I'm wondering...can sync really not fail anymore? @UdjinM6

Reply:
Well, governance objects/votes are still off-chain and smth might go wrong there... but we have no way of figuring it out so, technically, it's not going to fail :)